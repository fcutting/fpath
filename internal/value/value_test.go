package value

import (
	"testing"

	"github.com/gkampitakis/go-snaps/snaps"
)

func Test_ValueHolder_PutNumberValue(t *testing.T) {
	v := NewValueHolder()
	token := "123"
	value, err := v.PutNumberValue(token)

	if err != nil {
		t.Fatalf("Unexpected error: %s", err)
	}

	t.Run("NumberValues", func(t *testing.T) {
		snaps.MatchJSON(t, v.numberValues)
	})

	t.Run("Value", func(t *testing.T) {
		snaps.MatchJSON(t, value)
	})
}
