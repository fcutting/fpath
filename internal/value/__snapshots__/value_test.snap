
[Test_ValueHolder_PutNumberValue/NumberValues - 1]
{
 "0": "123"
}
---

[Test_ValueHolder_PutNumberValue/Value - 1]
{
 "Key": 0,
 "Type": 1
}
---
